.nvmrc

```diff
@@ -0,0 +1 @@
+6
```

Comment:
It means any version of Node 6 (e.g. 6.1.0 or 6.4.0, but not 7.0.0). Unfortunately nvm does not appear to support a value like `>= 6.0.0` 😢 

I'm happy to amend this PR to remove the .nvmrc file, since not everyone uses nvm, and the `engines` field of package.json is more robust.